qiskit/wrapper/defaultqiskitprovider.py

```diff
@@ -17,6 +17,34 @@
 logger = logging.getLogger(__name__)
 
 
+def _qiskit_supported_providers():
+    """Get the providers supported by the Qiskit team.
+
+    Returns:
+        list[class]: list of provider classes.
+    """
+    supported_providers = [LocalProvider]
+
+    # Add Qiskit-supported backends.
+    try:
```

Comment:
I was going to ... but on a second look, the semantic benefits would outweigh the drawbacks (the linter and tools not knowing that these particular modules are optional, and complaining quite a bit). Since the plan is to remove the `try` blocks entirely once the follow-up PR is completed, as mentioned in the description, I'll just add a `TODO` for these few days.